### PR TITLE
Try installing wasm-tools from a rollback file instead of from the feed

### PR DESCRIPTION
**Pull request by marcpopMSFT**

### CODE REVIEW

**Comment by marcpopMSFT** [comment]
No idea if this is going to work. When I tested locally, it created the json file in the right spot. I don't know if I got the versions right or if the update command in the build.yml will work. I copied the logic from installer repo for the workload version but I also picked the wasmtools version from there which is probably not the right one. Let's see what the PR build does and we can debug from there.

**Comment by marcpopMSFT** [comment]
@lewing  @dsplaisted any thoughts on this approach?

**Review by lewing — APPROVED** [review]
Looks good to me

**Review by lewing — APPROVED** [review]
Looks good to me

**Review by dsplaisted — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <VersionPrefix>8.0.100</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>5</PreReleaseVersionIteration>
+    <WorkloadsFeatureBand>8.0.100</WorkloadsFeatureBand>
```
[the remark this answers is not in the document]
Comment (marcpopMSFT) [reply eng/Versions.props:17]: I believe once a year as they stayed at 7.0.100 so far through all the 7.0 feature bands.

[the remark this answers is not in the document]
Comment (lewing) [reply eng/Versions.props:17]: even if we were to change this more it is well worth taking this pr

**Comment by dsplaisted** [comment]
Looks like there are some test failures, for example in `Microsoft.NET.Sdk.BlazorWebAssembly.AoT.Tests.WasmAoTPublishIntegrationTest.AoT_Publish_InRelease_Works`:

> Expected File "wwwroot/_framework/dotnet.wasm" cannot be found in directory "C:\h\w\A38108C9\t\dotnetSdkTests\1tmeru2k.50b\AoT_Publish_I---9BC42A2F\blazorwasm\bin\Release\net8.0\publish".

Could this be related to the fact that it's now using a different version of the workload?

**Comment by lewing** [comment]
@pavelsavara @maraf this is breaking because of the dotnet.wasm -> dotnet.native.wasm rename

**Comment by lewing** [comment]
@marcpopMSFT one thing that is clear is that the rollback hack should really apply for local testing too and at least all wasm tests

**Pull request merged**